examples/Python/ChatApp/app.py

```diff
@@ -0,0 +1,158 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+#
+import os
+import time
+from azure.identity import DefaultAzureCredential
+from azure.appconfiguration.provider import load, SettingSelector
+from openai import AzureOpenAI
+from models import ModelConfiguration, Message
+from typing import Dict, List, Any, Optional, Type, TypeVar
+
+# Get Azure App Configuration endpoint from environment variable
+ENDPOINT = os.environ.get("AZURE_APPCONFIG_ENDPOINT")
+if not ENDPOINT:
+    raise ValueError("The environment variable 'AZURE_APPCONFIG_ENDPOINT' is not set or is empty.")
+
+# Initialize Azure credentials
+credential = DefaultAzureCredential()
+
+# Create selector for ChatApp configuration
+chat_app_selector = SettingSelector(key_filter="ChatApp:*")
+
+# Load configuration from Azure App Configuration
+global config
+config = load(
+    endpoint=ENDPOINT,
+    selects=[chat_app_selector],
+    credential=credential,
```

Comment:
The API key should now be loaded from key vault, and the key vault credential is needed. It should be the same credential as for app configuration.